### PR TITLE
Introduce Sports, fix #94

### LINKED ISSUE #94: Add sport field to competition
It would be great to be able to flag a competition with a given sport and then associate it with some relevant icons we could display next to competition names.

### PR DESCRIPTION
**Pull request by LeBenLeBen**
- [x] Create Sport model, assigned to Competition
- [x] Add sport field to Competition form
- [x] Create current played Sports
- [x] Assign a sport to existing competitions
- [x] Allow sports creation/edition through the Admin
- [x] Display sport on competition page
- [x] Display an icon representing the sport in competition lists

### CODE REVIEW

File: apps/game/models/competition.py [hunk 1]
```diff
@@ -39,6 +39,15 @@ def get_queryset(self):
         return super().get_queryset().filter(start_date__gt=timezone.now())
 
 
+class Sport(models.Model):
+    slug = models.SlugField(unique=True)
+    name = models.CharField(max_length=255)
+    icon = models.TextField(null=True, blank=True)
```
Comment (sephii) [inline apps/game/models/competition.py]: Don't put `null=True` on text fields (https://docs.djangoproject.com/en/1.9/ref/models/fields/#null).

File: apps/game/templates/competition/detail.html [hunk 2]
```diff
@@ -8,9 +8,12 @@
             <div class="mirror__item mrgb--">
                 <h1 class="mrgb0">{{ competition }}</h1>
 
-                {% if competition.description %}
-                    <p class="mrgv0 text-muted">{{ competition.description | linebreaksbr }}</p>
-                {% endif %}
+                <ul class="list-inline list-inline--divided text-muted">
+                    <li>{{ competition.sport.icon|safe }} {{ competition.sport }}</li>
+                    {% if competition.description %}
+                        <li>{{ competition.description }}</li>
```
Comment (sephii) [inline apps/game/templates/competition/detail.html]: Is there a reason why you removed the `linebreaksbr` filter?

Comment (LeBenLeBen) [inline apps/game/templates/competition/detail.html]: Yeah the new display as an inline list make it difficult to display long descriptions with line breaks. According to its current usage I’m feeling like the description should be limited in characters length and be more like a text field than a text area. WDYT?

Comment (sephii) [inline apps/game/templates/competition/detail.html]: I agree, you should then probably change the type of `Competition.description` from `TextField` to `CharField`.

File: apps/game/models/competition.py (hunk 1 above)
Comment (LeBenLeBen) [inline apps/game/models/competition.py]: Fixed 👍

File: apps/game/templates/competition/detail.html (hunk 2 above)
Comment (LeBenLeBen) [inline apps/game/templates/competition/detail.html]: Done, changed the type and limited to 80 chars.

**Pull request merged**